integration/service/oasis/stationfinder/tnsearchfinder/tnsearch_station_finder.go

```diff
@@ -0,0 +1,34 @@
+package tnsearchfinder
```

Comment:
Nit: Can we remove the `tn`?

Reply:
Then would be two finders
- searchfinder
- localfinder

Let me think about it.  Or change to `cloudfinder`?

Reply:
How about `searchfinder` and `s2finder`?